osc/credentials.py

```diff
@@ -25,7 +27,6 @@
     print(msg, e, file=sys.stderr)
     gnomekeyring = None
 
-from . import conf
```

Comment:
The `conf` module is used later in the code, removing the line leads to the following errors found by pylint:
```
osc/credentials.py:234:42: E0602: Undefined variable 'conf' (undefined-variable)
osc/credentials.py:420:38: E0602: Undefined variable 'conf' (undefined-variable)
```